src/localforage.backbone.js

```diff
@@ -46,6 +46,24 @@
         return S4() + S4() + '-' + S4() + '-' + S4() + '-' + S4() + '-' + S4() + S4() + S4();
     }
 
+    function updateCollectionReferences(collection, callback, err, data) {
+        // If this model has a collection, keep the collection in =
+        // sync as well.
+        if (collection) {
+            // Create an array of `model.collection` models' ids.
+            var collectionData = collection.map(function(model) {
+                return collection.model.prototype.sync._localforageNamespace + '/' + model.id;
+            });
+
+            // Bind `data` to `callback` to call after
+            // `model.collection` models' ids are persisted.
+            callback = callback ? _.partial(callback, err, data) : void 0;
```

Comment:
@tofumatt this was existing code I've extracted from `save`. I'll update using `undefined`.